math/minuit2/inc/Minuit2/SinParameterTransformation.h

```diff
@@ -24,15 +25,21 @@ class MnMachinePrecision;
 class SinParameterTransformation {
 
 public:
+
    SinParameterTransformation() {}
 
    ~SinParameterTransformation() {}
 
-   double Int2ext(double Value, double Upper, double Lower) const;
-   double Ext2int(double Value, double Upper, double Lower, const MnMachinePrecision &) const;
-   double DInt2Ext(double Value, double Upper, double Lower) const;
+   long double Int2ext(long double Value, long double Upper, long double Lower) const;
```

Comment:
Does this affect the performance of those trafos? `long double` will be software emulated on most machines if I'm not mistaken.